lib/plugins.ts

```diff
@@ -508,7 +509,10 @@ export class Plugins {
               .map(
                   pluginObj =>
                       this.safeCallPluginFun(pluginObj, funName, args, promiseType, failReturnVal));
-      return promiseType == PromiseType.Q ? q.all(promises) : webdriver.promise.all(promises);
+      return promiseType == PromiseType.Q ?
```

Comment:
I'm a little ignorant here, do we need to differentiate between Q and native promises here? Can't `Promise.all` handle both types?